README.md

```diff
@@ -54,9 +54,19 @@ npm install -g oav
 ```
 bash-3.2$ oav -h
 Commands:
-  validate-example <spec-path>  Performs validation of x-ms-examples and
-                                examples present in the spec.
-  validate-spec <spec-path>     Performs semantic validation of the spec.
+  extract-xmsexamples <spec-path>           Extracts the x-ms-examples for a
```

Comment:
I wonder if we should re-order to show the validate commands that we expect people to use most, first.